screeninfo/enumerators/windows.py

```diff
@@ -36,20 +39,34 @@ def callback(monitor: T.Any, dc: T.Any, rect: T.Any, data: T.Any) -> int:
         else:
             name = None
 
+        h_size = ctypes.windll.gdi32.GetDeviceCaps(dc, HORZSIZE)
+        v_size = ctypes.windll.gdi32.GetDeviceCaps(dc, VERTSIZE)
+
         rct = rect.contents
         monitors.append(
             Monitor(
                 x=rct.left,
                 y=rct.top,
                 width=rct.right - rct.left,
                 height=rct.bottom - rct.top,
+                width_mm=h_size,
+                height_mm=v_size,
                 name=name,
             )
         )
         return 1
 
+    # Make the process DPI aware so it will detect the actual
+    # resolution and not a virtualized resolution reported by
+    # Windows when DPI virtualization is in use.
+    ctypes.windll.user32.SetProcessDPIAware()
+    # Create a Device Context for the full virtual desktop.
+    dc_full = ctypes.windll.user32.GetDC(None)
+    # Call EnumDisplayMonitors with the non-NULL DC
+    # so that non-NULL DCs are passed onto the callback.
+    # We want monitor specific DCs in the callback.
     ctypes.windll.user32.EnumDisplayMonitors(
-        0, 0, MonitorEnumProc(callback), 0
-    )
+        dc_full, None, MonitorEnumProc(callback), 0
+        )
```

Comment:
Wrong indentation

Reply:
Fixed this with e70f1fd, hopefully I got it right now.